src/kadai/Ma1.java

```diff
@@ -0,0 +1,28 @@
+package kadai;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Ma1 {
+
+    public static void main(String[] args) throws Exception { //throws Exception 例外処理
+
+       // 6.（自由課題）任意のプログラムで例外を起こしてみる。
+
+        Map< Integer, String > map = new HashMap<>(); // HashMap<キーのデータ型,値のデータ型> 変数名 = new HashMap<>();
+
+        int countnumber = 6 ; // 要素の数を代入させる　throws Exception で数が多いとnull出力
+
+        map.put( 1, " MAP1 " );
+        map.put( 2, " MAP2 " );
+        map.put( 3, " MAP3 " );
+        map.put( 4, " MAP4 " );
+        map.put( 5, " MAP5 " );
```

Comment:
見やすいようにコードを整頓してくれている意図が伝わります。
ありがとうございます！

少し確認したい点があります。

①こちら手でフォーマットされましたか？
IntelliJには自動フォーマット機能があります。
また、ファイル保存時に自動でフォーマットしてくれる機能もありますのでよければご活用ください。
`IntelliJ format`
`IntelliJ save actions`
といったキーワードで調べてみてください。

②mapの値である" MAP1 "ですが、こちら「MAP1」を意図されていますか？
だとしたら、以下のようにするとよいです。
```suggestion
        map.put(1, "MAP1");
        map.put(2, "MAP2");
        map.put(3, "MAP3");
        map.put(4, "MAP4");
        map.put(5, "MAP5");
```
ダブルクオートで囲んだ部分は半角スペースも文字の一部となりますので覚えておいてもらえるとありがたいです。

Reply:
質問はMAP1という出力の仕方が意図したものなのか？という意味でしょうか？
String型で文字列で間違いありませんが、スペースをいれるべき所、詰めるべき所という
基本的な書き方を覚えていくようにいたします。